lib/core/Head.js

```diff
@@ -16,13 +16,19 @@ class Head extends React.Component {
       if (link.blog) hasBlog = true;
     });
 
-    const highlightDefaultVersion = '9.12.0';
-    const highlightConfig = this.props.config.highlight || {
-      version: highlightDefaultVersion,
+    let highlight = {
+      version: '9.12.0',
       theme: 'default',
+      ...this.props.config.highlight,
     };
-    const highlightVersion = highlightConfig.version || highlightDefaultVersion;
-    const highlightTheme = highlightConfig.theme || 'default';
+
+    if (highlight.theme.indexOf('/') > -1) {
```

Comment:
This is not a very strong check. There's no guarantee that themes cannot have slashes in it in the future. Refer to comment below about using a `themeUrl` field as a check.